prototype/link/scripts/preprocessing_exploration/README.md

```diff
@@ -12,15 +12,15 @@ The main scripts that should be run in a sequence (can be run for multiple runs
 
 `prep_raw_data_explore.sh`: Creates all of the files needed to explore the parameters specified
 When running FEAT with the non-standard parameters (e.g. translational motion threshold equals 1mm) then you need to create the confound files that will make this possible. This script uses tools available from the `prep_raw_data.m` script to achieve this without overwriting the defaults.
-Specifically this makes parameters for all combinations of the following motion thresholds: 0.5mm, 1mm, 3mm, 6mm and 12mm, as well as these zipper detection thresholds: p=0.05, IQR and none (this is a legacy algorithm that attempts to detect 'zipper' artefacts that come from intraframe motion. It is ). These values are described in the variables `fslmotion_thresholds` and `PCA_thresholds`, respectively. 
+Specifically this makes parameters for all combinations of the motion thresholds as well as these zipper detection thresholds (set to just 0 now since this is a legacy algorithm that attempts to detect 'zipper' artefacts that come from intraframe motion.). These values are described in the variables `fslmotion_thresholds` and `PCA_thresholds`, respectively. 
 
 *Input/s*: functional run. e.g. `sbatch ./scripts/preprocessing_exploration/prep_raw_data_explore.sh 01`
 *Output/s*: Confound files in $SUBJ_DIR/analysis/firstlevel/Confounds/ for different preprocessing parameters of a run
 
 `preprocessing_exploration.sh`: Performs a FEAT analysis for all of the parameters specified.
-For each different type of preprocessing setting (listed in the variable `analysis_types`) this performs a FEAT analysis. A set of default parameters are defined (starting in the section `Set the default values`) and each analysis changes parameters relative to those parameters. The `default` analyses are those with no parameters changed are reported in text as the chosen parameters. These parameters are then used to create fsf files with the appropriate parameters and then the FEAT analysis is run.  
+For each different type of preprocessing setting (listed in the variable `analysis_types`) this performs a FEAT analysis. A set of default parameters are defined (starting in the section `Set the default values`) and each analysis changes parameters relative to those defaults. The `default` analyses, which with no parameters changed, are reported in text as the chosen parameters. These parameters are then used to create fsf files with the appropriate parameters and then the FEAT analysis is run.  
```

Comment:
The `default` analyses, which with no parameters changed, are reported in text as the chosen parameters. -----> strange wording. Maybe "The `default` analyses, which are those with no parameters changed, are reported in text as the chosen parameters"

Reply:
Yeah you're right, that was non-sensical